### PR TITLE
Hlm 729 | vc landing page-update

### PR DESCRIPTION
**Pull request by bhanuprakash-egov**
@prasanna-egov @saiprakash-v

### CODE REVIEW

File: tenant-portal/src/components/Home/Home.js [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
+import styles from './Home.module.css';
+import {useTranslation} from "react-i18next";
+import config from '../../config.json';
 
 function Home() {
 
     const navigate = useNavigate();
+    const { t } = useTranslation();
+    const genToken = () => {
+        navigate('/tenant-portal');
```
Comment (prasanna-egov) [inline tenant-portal/src/components/Home/Home.js:15]: can you use link?

File: tenant-portal/src/components/Home/Home.js [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
+    };
+    const manageSchema = () => {
+        navigate('/tenant-portal');
```
Comment (prasanna-egov) [inline tenant-portal/src/components/Home/Home.js:18]: same as above?

File: tenant-portal/src/components/Home/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.card-title{
```
Comment (prasanna-egov) [inline tenant-portal/src/components/Home/Home.module.css:1]: is this specific to Home page.

Comment (bhanuprakash-egov) [reply tenant-portal/src/components/Home/Home.module.css:1]: Yes, these styles are specific to home page

**Review by saiprakash-v — APPROVED** [review]

**Pull request merged**